### PR TITLE
ROX-23984: install vpa operator

### PR DESCRIPTION
**Pull request by ludydoo**
## Description
Installing the VPA operator.

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Added test description under `Test manual`
- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)
- [ ] CI and all relevant tests are passing
- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.
- [ ] Add secret to app-interface Vault or Secrets Manager if necessary
- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)
- [ ] Check AWS limits are reasonable for changes provisioning new resources
- [ ] (If applicable) Changes to the dp-terraform Helm values have been reflected in the addon on integration environment

## Test manual

**TODO:** Add manual testing efforts

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup
make verify lint binary test test/integration
```

### CODE REVIEW

**Review by ebensh — APPROVED** [review]
Looks good to me, though a couple questions:

1. Should we just install the operator as a normal openshift operator (via OCM) as part of our cluster turn-up, treating it like any other OpenShift provided service/dependency? If there's no configuration going on, why is it necessary to install via the Helm chart? And if we do want to install it via the Helm chart, can we document why in the PR description or a README.md, so we know why it's here?

Other than that the configuration looks reasonable to me

File: dp-terraform/helm/rhacs-terraform/charts/vertical-pod-autoscaler/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (ebensh) [inline dp-terraform/helm/rhacs-terraform/charts/vertical-pod-autoscaler/.helmignore:1]: Are these files really necessary?

Comment (ludydoo) [reply dp-terraform/helm/rhacs-terraform/charts/vertical-pod-autoscaler/.helmignore:1]: I don't think so, it was there for every chart, didn't want to break the convention
